Changelog.md

```diff
@@ -13,6 +13,7 @@
 ## FF7
 
 - Core: Add missing Cloud models in the model to eye mapping list
+- Rendering: Added option for japanese text rendering ( https://github.com/julianxhokaxhiu/FFNx/pull/737 )
```

Comment:
Can you please move this into the Next section?

Reply:
Done, somehow I always put it in the wrong place lol